2022-07-05 21:18:25,553 INFO    MainThread:9351 [internal.py:wandb_internal():90] W&B internal server running at pid: 9351, started at: 2022-07-05 21:18:25.553044
2022-07-05 21:18:25,554 DEBUG   HandlerThread:9351 [handler.py:handle_request():141] handle_request: run_start
2022-07-05 21:18:25,554 INFO    WriterThread:9351 [datastore.py:open_for_write():75] open: /home/mohan/git/current_projects/contextual_rl/attention/CARL/experiments/context_importance/outputs/2022-07-05/21-18-25/wandb/offline-run-20220705_211825-o2oi0cnv/run-o2oi0cnv.wandb
2022-07-05 21:18:25,556 DEBUG   HandlerThread:9351 [meta.py:__init__():35] meta init
2022-07-05 21:18:25,556 DEBUG   HandlerThread:9351 [meta.py:__init__():49] meta init done
2022-07-05 21:18:25,556 DEBUG   HandlerThread:9351 [meta.py:probe():209] probe
2022-07-05 21:18:25,562 DEBUG   HandlerThread:9351 [meta.py:_setup_git():199] setup git
2022-07-05 21:18:25,568 DEBUG   HandlerThread:9351 [meta.py:_setup_git():206] setup git done
2022-07-05 21:18:25,568 DEBUG   HandlerThread:9351 [meta.py:_save_pip():53] save pip
2022-07-05 21:18:25,568 DEBUG   HandlerThread:9351 [meta.py:_save_pip():67] save pip done
2022-07-05 21:18:25,568 DEBUG   HandlerThread:9351 [meta.py:_save_conda():74] save conda
2022-07-05 21:18:30,075 DEBUG   HandlerThread:9351 [meta.py:_save_conda():84] save conda done
2022-07-05 21:18:30,075 DEBUG   HandlerThread:9351 [meta.py:probe():247] probe done
2022-07-05 21:18:34,010 DEBUG   HandlerThread:9351 [handler.py:handle_request():141] handle_request: partial_history
2022-07-05 21:18:34,289 DEBUG   HandlerThread:9351 [handler.py:handle_request():141] handle_request: partial_history
2022-07-05 21:18:40,313 WARNING MainThread:9351 [internal.py:wandb_internal():153] Internal process interrupt: 1
2022-07-05 21:18:40,481 WARNING MainThread:9351 [internal.py:wandb_internal():153] Internal process interrupt: 2
2022-07-05 21:18:40,481 ERROR   MainThread:9351 [internal.py:wandb_internal():156] Internal process interrupted.
2022-07-05 21:18:40,556 INFO    SenderThread:9351 [sender.py:finish():1111] shutting down sender
2022-07-05 21:18:40,725 INFO    MainThread:9351 [internal.py:handle_exit():80] Internal process exited
